test/test_interactive.py

```diff
@@ -334,6 +334,20 @@ def test_audio_worklet_2x_hard_pan_io(self):
     shutil.copy(test_file('webaudio/audio_files/emscripten-bass-mono.mp3'), 'audio_files/')
     self.btest_exit('webaudio/audioworklet_2x_in_hard_pan.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'])
 
+  # Same as 'test_audio_worklet_2x_stereo_io' but as wasm64 with the worklet stack starting 2GB in (tests for unsigned shifts in the memory offsets)
+  def test_audio_worklet_stereo_io_2gb(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
+    self.btest_exit('webaudio/audioworklet_in_out_stereo.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMEMORY64', '-sINITIAL_MEMORY=2164260864', '-DTEST_OFFSET_GB=2'])
+
+  # Same as 'test_audio_worklet_stereo_io_2gb' but with a 4GB offset
+  def test_audio_worklet_stereo_io_4gb(self):
+    os.mkdir('audio_files')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-beat.mp3'), 'audio_files/')
+    shutil.copy(test_file('webaudio/audio_files/emscripten-bass.mp3'), 'audio_files/')
+    self.btest_exit('webaudio/audioworklet_in_out_stereo.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMEMORY64', '-sINITIAL_MEMORY=4311744512', '-DTEST_OFFSET_GB=4'])
```

Comment:
Can you use the same number we use in test_browser.py when defining `browser_2gb` and `browser64_4gb`?

i.e 2200mb or 4200mb for INITIAL_MEMORY and 2gb or 4gb for GLOBAL_BASE.

My setting GLOBAL_BASE that ensure that all stack and heap allocations are above that number, so you don't need to the test to be aware of any offset.   i.e. you can completely remove the TEST_OFFSET_GB thing I think

Reply:
Will do (which is better than what I was doing since I'm hoping it will break more things I spotted).